deploy/role.yaml

```diff
@@ -6,12 +6,18 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - secrets
   - pods
   verbs:
   - get
   - list
   - watch
+- apiGroups:
+    - ""
+  resources:
+    - secrets
+  verbs:
+    - get
+    - create
```

Comment:
I would suggest to add add `secrets` along with the `configmaps` resource, as the operator needs the delete permission.

Reply:
I believe we might want to have delete permissions when we will regenerate the certificate, or not ?

Reply:
At least it will need the `update` verb. For the `delete` as the Secret gets deleted by cascading ownership when the Hawtio resource gets deleted, I don't know if it is needed or not. I guess we have to test it.

Reply:
Ok so I tested it and secret was successfully deleted also without "delete" permission so I removed it

Reply:
Great, thanks for the confirmation.